packages/ui/molecules/Menu/Menu.js

```diff
@@ -0,0 +1,241 @@
+import { Base } from '@studiometa/js-toolkit';
+import { nextTick } from '@studiometa/js-toolkit/utils';
+import MenuBtn from './MenuBtn.js';
+import MenuItems from './MenuItems.js';
+
+/**
+ * @typedef {Menu & {
+ *   $children: {
+ *     Menu: Menu[],
+ *     MenuBtn: MenuBtn[],
+ *     MenuItems: MenuItems[],
+ *   }
+ * }} MenuInterface
+ */
+
+/**
+ * Menu class.
+ */
+export default class Menu extends Base {
+  /**
+   * Config.
+   */
+  static config = {
+    name: 'Menu',
+    debug: true,
+    components: {
+      MenuBtn,
+      MenuItems,
+      Menu,
+    },
+    options: {
+      mode: {
+        type: String,
+        default: 'click', // or 'hover'
+      },
+    },
+  };
+
+  /**
+   * Get direct children.
+   *
+   * @this    {MenuInterface}
+   * @param   {string} name
+   * @returns {any[]}
+   */
+  getDirectChildren(name) {
+    if (!this.$children[name]) {
+      return [];
+    }
+
+    if (!this.$children.Menu) {
+      return this.$children[name];
+    }
+
+    return this.$children[name].filter((child) =>
+      this.$children.Menu.every((menu) =>
+        menu.$children[name] ? !menu.$children[name].includes(child) : true
+      )
+    );
+  }
+
+  /**
+   * Get the first `MenuItems` instance.
+   *
+   * @this    {MenuInterface}
+   * @returns {MenuItems}
+   */
+  get menuItems() {
```

Comment:
**nitpick** rename to `menuFirstItem()` to reflect the function behavior ?

Reply:
Maybe the component should be renamed to avoid confusion, as we should only have one instance as a direct child.

I was thinking about `MenuList` or maybe `MenuContent`. Any preference or idea @studiometa/core?

Reply:
I switched to `MenuList`.